lib/bibdata_rs/src/ephemera/ephemera_folder.rs

```diff
@@ -98,6 +167,7 @@ impl EphemeraFolder {
     pub fn origin_place_labels(&self) -> Vec<String> {
         match &self.origin {
             Some(origin_vector) => origin_vector
+                .0
```

Comment:
I think we can remove `.0` here.  Since you implemented `Deref`, `origin_vector` has all the methods of `Vec` already available to it!  At least, that is what I understand from [the third bullet point here](https://doc.rust-lang.org/std/ops/trait.Deref.html#deref-coercion):

> If `T` implements `Deref<Target = U>`, [...] `T` implicitly implements all the methods of the type `U` which take the `&self` receiver.